packages/gafl-webapp-service/src/handlers/__tests__/check-analytics-handler.spec.js

```diff
@@ -1,10 +1,13 @@
 import { ANALYTICS } from '../../constants.js'
 import { CacheError } from '../../session-cache/cache-manager.js'
-import { checkAnalytics, getAnalyticsSessionId } from '../analytics-handler.js'
+import { checkAnalytics, getAnalyticsSessionId, checkPage } from '../analytics-handler.js'
 
 jest.mock('../../constants', () => ({
   ANALYTICS: {
-    acceptTracking: 'accepted-tracking'
+    selected: 'selected',
+    acceptTracking: 'accepted-tracking',
+    seenMessage: 'seen-message',
+    skipPage: 'skip-page'
```

Comment:
these should differ from what's in constants.js

Reply:
I think it might be that we added a separate test for this on line 30.